partiql-tests-data/eval/query/group-by/group-by.ion

```diff
@@ -1621,68 +1622,7 @@
       ],
       output:$bag::[
         {
-        }
-      ]
-    }
-  },
-  {
-    name:"SELECT groupExp FROM simple_1_col_1_group GROUP BY NULL AS groupExp",
-    statement:"SELECT groupExp FROM simple_1_col_1_group GROUP BY NULL AS groupExp",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-          groupExp:null
-        }
-      ]
-    }
-  },
-  {
-    name:"SELECT VALUE { 'groupExp': groupExp } FROM simple_1_col_1_group GROUP BY NULL AS groupExp",
-    statement:"SELECT VALUE { 'groupExp': groupExp } FROM simple_1_col_1_group GROUP BY NULL AS groupExp",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-          groupExp:null
-        }
-      ]
-    }
-  },
-  {
-    name:"SELECT groupExp FROM simple_1_col_1_group GROUP BY MISSING AS groupExp",
-    statement:"SELECT groupExp FROM simple_1_col_1_group GROUP BY MISSING AS groupExp",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
-        }
-      ]
-    }
-  },
-  {
-    name:"SELECT VALUE { 'groupExp': groupExp } FROM simple_1_col_1_group GROUP BY MISSING AS groupExp",
-    statement:"SELECT VALUE { 'groupExp': groupExp } FROM simple_1_col_1_group GROUP BY MISSING AS groupExp",
-    assert:{
-      result:EvaluationSuccess,
-      evalMode:[
-        EvalModeCoerce,
-        EvalModeError
-      ],
-      output:$bag::[
-        {
```

Comment:
These ported tests were nearly identical to the previous tests other than the group key alias. Removed in this PR due to this redundancy.